WebDriverAgentLib/Utilities/FBScreenshot.m

```diff
@@ -212,7 +212,7 @@ + (BOOL)shouldUseScreenshotRequestApiForProxy:(NSObject *)proxy
   dispatch_once(&shouldUseSRApi, ^{
     if ([proxy respondsToSelector:@selector(_XCT_requestScreenshot:withReply:)]) {
 #if TARGET_OS_SIMULATOR
-      result = YES;
+      result = SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"14.5");
```

Comment:
could you please add the explanation comment here as well?

Reply:
Done.